include/net/gptp.h

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Public functions for the Precision Time Protocol Stack.
+ *
+ */
+
+#ifndef __GPTP_H
+#define __GPTP_H
+
```

Comment:
Need to define a doxygen defgroup to have this API show up in the documentation (and need a reference added to one of the .rst files in doc/api)